_includes/04-lists.md

```diff
@@ -0,0 +1,3 @@
+Emoji
+Ordered Lists
+Unordered Lists
```

Comment:
Unordered Lists
To create an unordered list, use either the - or * character. As with ordered lists, you can nest a list by indenting two spaces.

* Item 1
* Item 2
  * Item 2a
  * Item 2b
Item 1
Item 2
Item 2a
Item 2b